audio/src/backend/openal.cpp

```diff
@@ -135,7 +135,7 @@ qreal OpenAL::outputVolume() const
         checkAlError();
     }
 
-    return volume;
+    return static_cast<qreal>(volume);
```

Comment:
You could also change the return type of this function since this is the only implementation. ...or delete it altogether since it looks like it's completely unused?

Reply:
Removed it completely.